packages/sources/finage/src/endpoint/uk-etf.ts

```diff
@@ -9,4 +9,4 @@ export const inputParameters = etfInputParameters
 export const description = `https://finage.co.uk/docs/api/etf-last-price
 The result will be the price field in response.`
 
-export const execute = makeEtfExecute({ country: 'uk' })
```

Comment:
The `uk-etf` specific endpoint should default to `UK` if no country code is provided as an input param